Pester.psm1

```diff
@@ -221,6 +230,85 @@ about_pester
     if ($EnableExit) { Exit-WithCode -FailedCount $pester.FailedCount }
 }
 
+function ResolveTestScripts
+{
+    param ([object[]] $Path)
```

Comment:
I'm okay with that, but we'd need to keep a Path alias to maintain backward compatibility.  Maybe something like `-Script` or `-TestScript`?